lib/langAssembler.js

```diff
@@ -15,6 +15,14 @@ module.exports.assemble = assemble;
 function assemble(callback) {
     var localeDirectory = 'lang';
 
+    try {
+        Fs.statSync(localeDirectory);
+    }
+    catch (e) {
+        // No lang folder, callback immediately
+        return callback(null, 'no_lang_folder');
```

Comment:
It's not really an error though, everything can still proceed, but it's there to provide an informational alert. An error should only be for something that keeps it from bundling.